src/Serializer/AbstractItemNormalizer.php

```diff
@@ -68,7 +68,7 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
                 return $this->iriConverter->getIriFromItem($object);
             };
         }
-        if (!interface_exists(AdvancedNameConverterInterface::class)) {
+        if (!interface_exists(AdvancedNameConverterInterface::class) && method_exists($this, 'setCircularReferenceHandler')) {
```

Comment:
Hmm... I'm confused about this one. The interface exists in `symfony/serializer` 5.0, so this method isn't being called. It's a real false positive in phpstan.